kokoro/windows/build.bat

```diff
@@ -24,10 +24,11 @@ set JAVA_HOME=c:\Program Files\Java\jdk1.8.0_144
 
 REM Install the Android SDK components and NDK.
 set ANDROID_HOME=%LOCALAPPDATA%\Android\Sdk
-echo y | %ANDROID_HOME%\tools\bin\sdkmanager build-tools;26.0.1 platforms;android-26
-wget -q https://dl.google.com/android/repository/android-ndk-r18b-windows-x86_64.zip
-unzip -q android-ndk-r18b-windows-x86_64.zip
-set ANDROID_NDK_HOME=%CD%\android-ndk-r18b
+copy /Y "%SRC%\kokoro\windows\android-sdk-license" "%ANDROID_HOME%\licenses\"
+call %ANDROID_HOME%\tools\bin\sdkmanager.bat platforms;android-26 build-tools;29.0.2
```

Comment:
This seems like it's not flexible - why not use the `echo y` anymore?

Reply:
echo y doesn't work with sdkmanager.bat, and I guess it never did. It didn't cause problems before because build-tools 26.0.0 were already installed.

Reply:
An alternative would be to use yes.exe from mingw if we change the installation order, but it was also triggering a warning that might cause problems in the future.